tools/Documentation/plugin-docs/login.md

```diff
@@ -65,18 +64,17 @@ sub plugin_info {
 sub do_login {
 
     # Login plugins only receive the parameters entered by the user.
-    shift;
-    my ( $loginenabled, $uID, $password) = @_;
+    my ( undef, $params ) = @_;
 
     my $logger = get_logger( "Undernet Login", "plugins" );
     my $ua = Mojo::UserAgent->new;
 
-    if ($loginenabled) {
+    if ($params->{loginenabled}) {
 
         $ua->cookie_jar->add(
             Mojo::Cookie::Response->new(
                 name   => 'userID',
-                value  => $uID,
+                value  => $params->{uID},
```

Comment:
Are hash keys case-sensitive? I don't believe so but it might be better to show the same casing between parameter declaration and usage in the examples.